lib/js/components/Badges/Badge/Badge.stories.ts

```diff
@@ -0,0 +1,39 @@
+// Badge.stories.ts
+
+import Badge from './Badge.vue';
+import { BADGE_COLORS } from './Badge.consts';
+
+import { Meta, Story } from '@storybook/vue';
+
+export default {
+	title: 'Components/Badges/Badge',
+	component: Badge,
+} as Meta<typeof Badge>;
+
+const Template: Story<{ label: string; iconUrl: string; color: Object }> = (argTypes) => ({
```

Comment:
is it ok that we call it `Template` while it's `Story` type?

Reply:
Hmm so how we should name it? Like this?
```
const InteractiveStory: Story<....
...
export const Interactive = InteractiveStory.bind({});

```

Reply:
maybe `StoryTemplate` or `AbstractStory` from OOP?

Reply:
`StoryTemplate` sounds well.